src/pytorch_lightning/callbacks/progress/rich_progress.py

```diff
@@ -597,3 +605,16 @@ def _detect_light_colab_theme() -> bool:
         except ModuleNotFoundError:
             return False
     return False
+
+
+def _is_jupyter() -> bool:
```

Comment:
We should probably use our `_IS_INTERACTIVE` utility :)

Reply:
Then I would rather make a utility that detects colab specifically. Did you also check what happens in the interactive Python shell?